django_project/lesson/models/curriculum_worksheets.py

```diff
@@ -0,0 +1,34 @@
+# coding=utf-8
+
+"""Curriculum model definitions for lesson apps."""
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from lesson.models.curriculum import Curriculum
+from lesson.models.worksheet import Worksheet
+
+
+class CurriculumWorksheets(models.Model):
```

Comment:
I wonder why we can't just use a manytomany field in the curriculum model. See https://docs.djangoproject.com/en/2.0/topics/db/examples/many_to_many/

Reply:
Correct me if I'm wrong, but we can't because we have the sequence number field on the relationship.
https://docs.djangoproject.com/en/dev/topics/db/models/#extra-fields-on-many-to-many-relationships

Reply:
Ah good point - LGTM then!